readthedocs/core/views/__init__.py

```diff
@@ -45,15 +44,26 @@ class HomepageView(TemplateView):
     template_name = 'homepage.html'
 
     def get(self, request, *args, **kwargs):
+        # Redirect to login page for new dashboard
         if settings.RTD_EXT_THEME_ENABLED:
             return redirect(reverse("account_login"))
-        return super().get(request, *args, **kwargs)
 
-    def get_context_data(self, **kwargs):
-        """Add latest builds and featured projects."""
-        context = super().get_context_data(**kwargs)
-        context['featured_list'] = Project.objects.filter(featured=True)
-        return context
+        # Redirect to user dashboard for logged in users
+        if request.user.is_authenticated:
+            return redirect("projects_dashboard")
+
+        # Redirect to ``about.`` in production
+        if not settings.DEBUG:
+            query_string = "?ref=dotorg-homepage"
```

Comment:
I'm guessing we don't use this view on the commercial application, as we are already adding this to the requests on commercial. But if we do, this query string could be conditional based on the application instance.

Reply:
Yea, we're overriding it. Could probably stop now though, if we wanted 🤔